kyuubi-hive-jdbc/pom.xml

```diff
@@ -30,6 +30,12 @@
     <name>Kyuubi Project Hive JDBC Client</name>
     <packaging>jar</packaging>
 
+    <properties>
+        <!-- only used in kyuubi-hive-jdbc module -->
+        <junit.version>4.11</junit.version>
+        <mockito-all.version>1.9.5</mockito-all.version>
+    </properties>
```

Comment:
cc @pan3793

Reply:
Let's put them in root pom.xml L146, and also start with hive.jdbc.